### PR TITLE
Game settings

### PR DESCRIPTION
**Pull request by RobynGebbie**
#2 
#3 
Issue 2: This creates the settings page, rerouting to the settings page from a home screen. The user has the option of selecting whether they want the gram to be in dark mode or light mode. The current mode is loaded from the database and implemented in the game.

Issue 3: This allows the user to change the game settings. The user can select whether they want the game to be in dark mode or light mode. This value is saved in the database and the page is updated on refresh.

### CODE REVIEW

File: settings.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<body>
+    <h1 id="heading">Game Settings</h1>
+    <p id="results" style="font-size: 15px; font-weight: bold;">
+    </p>
+    <button type="button" class="btn btn-primary" id="darkButton">Dark Mode</button>
+    <button type="button" class="btn btn-primary" id="lightButton">Light Mode</button>
+    <button type="button" class="btn btn-primary" id="backButton">Back</button>
+    <p id="status"></p>
+    <p id="msgFmServer"></p>
+    <script defer src="/public/settings.js"></script>
```
Comment (UntidyRAM) [inline settings.html:24]: This needs to go to line 12.

File: public/fetchMode.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    helloButton.style.color = textColour
+    settingsButton.style.backgroundColor = buttonColour
+    settingsButton.style.color = textColour
+
+    // set current current scheme variables
+    sessionStorage.setItem('textColour', textColour)
+    sessionStorage.setItem('backColour', backColour)
+    sessionStorage.setItem('buttonColour', buttonColour)
+  })
+  .catch(error => document.write(`Could not fetch verse: ${error}`))
```
Comment (UntidyRAM) [inline public/fetchMode.js:52]: What's a verse? Is this maybe a typo?

**Review by UntidyRAM — APPROVED** [review]
It looks good. There's just one or two small things that need to be fixed _before_ merging :)

File: public/sendToServer.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  // send request to server
+  request.setRequestHeader('Content-type', 'application/json')
+
+  // convert the dark mode setting to a JSON object and send it to the server.
+  request.send(JSON.stringify({ darkMode }))
+
+  request.addEventListener('load', dataReceived)
+
+  const statusTag = document.getElementById('status')
+  statusTag.innerHTML = 'Status: Sent word to server'
```
[the remark this answers is not in the document]
Comment (RobynGebbie) [reply public/sendToServer.js:19]: This should be mode, will change

File: settings.html (hunk 1 above)
Comment (RobynGebbie) [reply settings.html:24]: Changed

File: public/fetchMode.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    if (darkMode === 'true') {
+      textColour = 'blue'
+      backColour = 'black'
+      buttonColour = 'white'
+    } else {
+      textColour = 'black'
+      backColour = 'white'
+      buttonColour = 'grey'
+    }
+
```
[the remark this answers is not in the document]
Comment (RobynGebbie) [reply public/fetchMode.js:30]: Will find a solution in next sprint

**Pull request merged**